lib/workers/branch/npmrc.js

```diff
@@ -0,0 +1,17 @@
+const ini = require('ini');
+
+module.exports = { validateNpmrc };
+
+function validateNpmrc(input) {
+  const npmrc = ini.parse(input);
```

Comment:
What happens if `ini` fails to parse?

Reply:
https://github.com/dominictarr/config-chain/blob/0b6db3e14b9cdbe31460292bc4caf3983f977816/index.js#L48

npm doesn't catch `ini.parse()`, should renovate?